x/compute/client/cli/tx.go

```diff
@@ -176,18 +192,20 @@ func InstantiateContractCmd(cdc *codec.Codec) *cobra.Command {
 
 			// build and sign the transaction, then broadcast to Tendermint
 			msg := types.MsgInstantiateContract{
-				Sender:    cliCtx.GetFromAddress(),
-				Code:      codeID,
-				Label:     label,
-				InitFunds: amount,
-				InitMsg:   initMsg,
-				Admin:     adminAddr,
+				Sender:           cliCtx.GetFromAddress(),
+				CallbackCodeHash: "",
```

Comment:
I probably don't understand this context completely, but - why is `CallbackCodeHash` here empty?